frontend/common/src/liquity/index.ts

```diff
@@ -31,11 +32,32 @@ export type LiquityStakingDetails = z.infer<typeof LiquityStakingDetails>;
 export const LiquityPoolDetail = z.object({
   gains: AssetBalance,
   rewards: AssetBalance,
-  deposited: AssetBalance
+  deposited: AssetBalance,
+  proxyTo: z.string().nullish()
 });
 
 export type LiquityPoolDetail = z.infer<typeof LiquityPoolDetail>;
 
 export const LiquityPoolDetails = z.record(LiquityPoolDetail);
 
 export type LiquityPoolDetails = z.infer<typeof LiquityPoolDetails>;
+
+export const LiquityStatistic = z.object({
```

Comment:
Maybe it would make sense to use `LiquityStatistics` here as a name, singular feels a bit wrong here, but we will have to change the name of the current type.